shuup/front/utils/dashboard.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2016, Shoop Commerce Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from django.core.urlresolvers import reverse
+from django.template.loader import get_template
+from django.utils.safestring import mark_safe
+from django.utils.translation import ugettext_lazy as _
+from enumfields import Enum
+
+
+class DashboardItemSize(Enum):
+    SMALL = 6
+    MEDIUM = 9
+    LARGE = 12
+
+
+class DashboardItem(object):
+    request = None
+    template = "shuup/front/dashboard/dashboard_base.jinja"
+    title = _("Dashboard")
+    size = DashboardItemSize.LARGE
+    icon = "fa fa-tachometer"
+    view_text = _("Edit")
+
+    _url = "shuup:dashboard"
+
+    def __init__(self, request):
+        self.request = request
+
+    @property
+    def url(self):
+        return reverse(self._url)
+
+    def get_context(self):
+        return {"item": self, "request": self.request}
+
+    def render(self):
+        template = get_template(self.template)
+        return mark_safe(template.render(context=self.get_context()))
+
+    @property
+    def css_class(self):
+        return "col-md-%d" % self.size.value
+
+    def show_on_menu(self):
+        """
+        Defines if the item is shown in the main dashboard menu
```

Comment:
Also the rest of the properties should have docstrings. Also this one seems to be missing something from return.